kitsune/wiki/utils.py

```diff
@@ -194,3 +212,51 @@ def get_visible_document_or_404(
 
 def get_visible_revision_or_404(user, **kwargs):
     return get_object_or_404(Revision.objects.visible(user, **kwargs))
+
+
+def build_topics_data(request, product, topics):
+    """Build topics_data for use in topic cards"""
+    topics_data = []
+
+    # Get all documents for all topics up front
+    all_docs, _ = documents_for(
+        request.user, request.LANGUAGE_CODE, topics=list(topics), products=[product]
```

Comment:
Why is the `list()` conversion needed? It looks like `_documents_for` expects an iterable and not a list specifically. This will also force the evaluation of the queryset.